### ISSUE TITLE
[SFSF][A11y][MultiComboBox] Focus is lost after deleting the tokens and notification message is being read out by screen readers

### ISSUE THREAD

**Issue by saorabhkr** [SAP/ui5-webcomponents#6671]
### Describe the bug

Describe the bug

With MultiComboBox, when we delete any tokens we have observed two things: 

> After deleting any token focus is not staying there on the previous/next token.

> Upon deletion screen reader is not announcing "Token deleted" or anything. 

The same can be seen here:
https://sap.github.io/ui5-webcomponents-react/?path=/story/inputs-multicombobox--default-story

### Isolated Example

https://sap.github.io/ui5-webcomponents-react/?path=/story/inputs-multicombobox--default-story

### Reproduction steps

Go to [Demo Page](https://sap.github.io/ui5-webcomponents-react/?path=/story/inputs-multicombobox--default-story)

Select a few tokens from the dropdown
Delete the tokens
It's been observed that it's not retaining the focus.
Now delete any token and the announcement by NVDA/JAWS
It's not announcing anything.

### Expected Behaviour

MultiComboBox:

> After deleting any token focus should stay on the previous/next token

> After deleting any token screen reader should announce some message eg: "Token deleted"
on

### UI5 Web Components for React Version

1.8.1

### UI5 Web Components Version

1.10.5

### Browser

Chrome

### Operating System

MacOS

**Comment by plamenivanov91** [comment]
Hello @SAP/ui5-webcomponents-topic-rl ,

Currently if we focus a token in the [MCB react default story](https://sap.github.io/ui5-webcomponents-react/?path=/story/inputs-multicombobox--default-story) and delete it with Backspace or Delete button, the tokens preview is closed (it says two more if there are two tokens for example).

The reporter claims that instead of closing the editing of the tokens, it should transfer the focus to the next/previous token and when we delete a Token, the screen reader should notify us about that action.

Are those claims valid, or the MCB works that way by design?

Could you please take over?

Regards,
Plamen Ivanov

**Comment by saorabhkr** [comment]
Hi @Lukas742,

> After deleting any token focus is not staying there on the previous/next token.
Please see the attached video, captured from storybook 

https://user-images.githubusercontent.com/5204425/223921369-2b18abac-83f8-45f8-88ee-6ff11a7b0126.mov

Also created an isolated example.
https://codesandbox.io/s/adoring-frost-pt0yit?file=/src/App.js 

Please let me know if you need any other details.

Best Regards, 
Saorabh

**Comment by saorabhkr** [comment]
Hi @Lukas742, 

We have found one more issue, with JAWS placeholder for MultiComboBox is not being read out. It should announce placeholder text with JAWS screen reader. 

Best Regards, 
Saorabh
